controllers/productController.js

```diff
@@ -0,0 +1,64 @@
+const prodcutService = require('../services/productService');
+
+const createReview = async (req, res) => {
+    try {
+        const { content, userIdCode, productId } = req.body;
+
+        if (!content || !userIdCode || !productId) {
+            return res.status(404).json({ message: "KEY_ERROR" });
```

Comment:
```suggestion
            return res.status(400).json({ message: "KEY_ERROR" });
```
`key error`의 경우 400을 반환해 주세요.
`404 status code`에 대해 조금 구글링해 보시면 좋을 것 같아요!

현재 PR의 `Files changed`를 직접 훑어보시고 전부 수정해주세요.